packages/webdriverio/src/commands/element/scrollIntoView.ts

```diff
@@ -2,9 +2,127 @@ import logger from '@wdio/logger'
 import { ELEMENT_KEY } from 'webdriver'
 
 import { getBrowserObject } from '@wdio/utils'
+import type { CustomScrollIntoViewOptions, MobileScrollIntoViewOptions } from 'src/types.js'
 
 const log = logger('webdriverio')
 
+type MobileScrollUntilVisibleOptions = {
```

Comment:
Can we move the code down, below the command function? Having other types or functions defined before just invites to add comments to them which will ultimately replace the command documentation when generating the docs

Reply:
sure, I added it to the top due to coding conventions that we first need to declare before we use